Cargo.toml

```diff
@@ -39,6 +39,21 @@ name = "fs_write_also_read"
 path = "tests/fs_write_also_read.rs"
 harness = false
 
+[[test]]
+name = "fs_symlink"
+path = "tests/fs_symlink.rs"
+harness = false
+
+[[test]]
+name = "fs_symlink_dir"
+path = "tests/fs_symlink_dir.rs"
+harness = false
```

Comment:
Initially I thought the issue was with symlinks, so I wrote two tests to check if symlinks work or not. Turns out they work just fine, but I thought I might as well keep the tests around.

These tests do leak a couple files to `/tmp`, since we can't clean up after locking down the sandbox, but I don't think that should be an issue (same applies to fs tests).